readthedocsext/theme/templates/profiles/base_edit.html

```diff
@@ -21,8 +21,16 @@
              href="{% url 'profiles_profile_edit' %}">{% trans "Account" %}</a>
           <a class="item {% block profile_admin_change_email %}{% endblock %}"
              href="{% url 'account_email' %}">{% trans "Email addresses" %}</a>
-          <a class="item {% block profile_admin_security %}{% endblock %}"
-             href="{% url 'account_change_password' %}">{% trans "Security" %}</a>
+
+          <div class="item {% block profile_admin_security %}{% endblock %}">
+            <div>{% trans "Security" %}</div>
```

Comment:
This should be a `header` element

```suggestion
            <div class="header">{% trans "Security" %}</div>
```

Reply:
With this class, the text is always in bold, even when no selected, are we okay with that?

![Screenshot from 2024-10-28 14-24-19](https://github.com/user-attachments/assets/10fa4f75-ec44-45fb-b9dd-dd79157e880f)

Reply:
Yes, this is how the project admin looks now too.

![image](https://github.com/user-attachments/assets/82633eb5-d092-4c09-a147-dc4b143867d1)

Eventually all options should be grouped like the project admin settings. If you'd like to match, you could throw all other options under a "General" header for now, or split them up more.

Reply:
What about

![Screenshot 2024-10-28 at 16-15-34 Connected services - Read the Docs](https://github.com/user-attachments/assets/1d97418a-dfec-4319-aa5d-e422da7c3433)

I'd say that security log and api tokens should also be in the security section.

Reply:
Also, with this change, do we still want to have the category as main header on each option page? e.g (Settings\nTwo-factor authentication)

Reply:
Yeah, I'd agree on API tokens and security log under `Security` header.

> Also, with this change, do we still want to have the category as main header on each option page?

For the 2fa page, this makes sense. I'd alter the main heading and sub header breadcrumbs a little bit in order to make this page more top level, and would aim for:

![image](https://github.com/user-attachments/assets/bfdb8dda-581c-43a0-92e3-a9d82a597ca5)

Where the first breadcrumb links back to the main page.

I'm aiming to eventually standardize the breadcrumb sub header on all pages.